### PR TITLE
Noto Serif Vithkuqi: Version 1.005 added

### PR DESCRIPTION
**Pull request by simoncozens**
2c5baa2: [gftools-packager] Noto Serif Vithkuqi: Version 1.005 added

* Noto Serif Vithkuqi Version 1.005 taken from the upstream repo https://github.com/notofonts/vithkuqi.git at commit https://github.com/notofonts/vithkuqi/commit/.

### CODE REVIEW

File: ofl/notoserifvithkuqi/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1 @@
+N/A
```
Comment (RosaWagner) [inline ofl/notoserifvithkuqi/DESCRIPTION.en_us.html:1]: @simoncozens missing description

File: ofl/notoserifvithkuqi/METADATA.pb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+axes {
+  tag: "wght"
+  min_value: 400.0
+  max_value: 700.0
+}
+source {
+  repository_url: "https://github.com/notofonts/vithkuqi.git"
+  archive_url: "https://github.com/notofonts/vithkuqi/releases/download/NotoSerifVithkuqi-v1.005/NotoSerifVithkuqi-v1.005.zip"
+}
+is_noto: true
```
Comment (RosaWagner) [inline ofl/notoserifvithkuqi/METADATA.pb:27]: @simoncozens missing subset, languages and primary script.

**Comment by simoncozens** [comment]
Done, waiting on merge of https://github.com/googlefonts/lang/pull/88

File: ofl/notoserifvithkuqi/upstream.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+archive: https://github.com/notofonts/vithkuqi/releases/download/NotoSerifVithkuqi-v1.005/NotoSerifVithkuqi-v1.005.zip
+branch: main
+files:
+  OFL.txt: OFL.txt
+  NotoSerifVithkuqi/googlefonts/variable-ttf/NotoSerifVithkuqi[wght].ttf: NotoSerifVithkuqi[wght].ttf
```
Comment (RosaWagner) [inline ofl/notoserifvithkuqi/upstream.yaml:5]: @simoncozens this one is still missing description and article

**Pull request merged**